packages/blade/src/components/Button/BaseButton/getBaseButtonStyles.ts

```diff
@@ -0,0 +1,31 @@
+import type { StyledBaseButtonProps } from './StyledBaseButton';
+
+const getBaseButtonStyles = ({
+  color,
+  hoverColor,
+  activeColor,
+}: Omit<StyledBaseButtonProps, 'children' | 'onClick'>): string => `
+  display:flex;
+  flex-direction: row;
+  height: 48px;
```

Comment:
should be `min-height` else the zooming and default font size change will break layouts. Check this [demo app](https://codesandbox.io/s/szi8i?file=/src/styles.css)  from [RFC](https://github.com/razorpay/blade/blob/master/rfcs/2021-02-19-units-for-typography-and-layout.md#what-will-work-for-us)